soccer/src/soccer/tutorial/soccer_mom_node.cpp

```diff
@@ -0,0 +1 @@
+#include "soccer_mom_node.hpp"
```

Comment:
It's usually better practice to write the functionality of your project in the source file rather than the header file. Since this was a small task, it should be fine.